### DISCUSSION TITLE
Console TransferAllClaims

### DISCUSSION THREAD

**Discussion by LoSCraft-Network** [GriefPrevention/GriefPrevention#844] [Ideas]
A command to transfer all claims from one player to another (Including replacing the player that is having it's claims transferred to 0 total of claim blocks, and the one receiving the claims to the amount of the other player) 

This would be really useful for non-premium servers, when users buy premium accounts and want to transfer everything to their new account, it is a pain to do.

**An example of the situation:**
Dandan has **3** claims and **1000** claim blocks 

Dandan bought a premium minecraft account with the name of **Dandan220**

When I run through the console or in-game the command:
**/transferallclaims Dandan Dandan220**

Dandan loses all claims and claim blocks (everything is set to 0) and Dandan220 receives all claims and gets the amount of claim blocks that he had on his old account (Dandan)

**Comment by RoboMWM** [comment]
Well, that's the pain of having your server set to offline mode in the first place...

Anyways, this feature exists in some form in GPRealEstate (my forked version)
